charts/cofide-connect/templates/deployment.yaml

```diff
@@ -17,11 +16,7 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       labels:
-        app.kubernetes.io/name: {{ .Release.Name }}
-        {{- include "cofide-connect.labels" . | nindent 8 }}
-        {{- with .Values.podLabels }}
-        {{- toYaml . | nindent 8 }}
-        {{- end }}
```

Comment:
I'm wondering what was wrong with this. Wouldn't it be better to keep using labels & values.podLabels here?